.lintrunner.toml

```diff
@@ -1477,6 +1477,7 @@ init_command = [
     'black==23.12.1',
     'usort==1.0.8.post1',
     'isort==5.13.2',
```

Comment:
Won't ruff fmt also handle isort's job too?

Reply:
`ruff check --select=I --fix` do not respect `# isort: split` for asterisk imports.

```python
from xxx.b import *  # isort: split
from xxx.a import *  # isort: split  # avoid circular imports
```

will be format to:

```python
from xxx.a import *  # isort: split  # avoid circular imports
from xxx.b import *  # isort: split
```

Reply:
FYI, https://github.com/astral-sh/ruff/issues/11776#issuecomment-2164181052